go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/expr-lang/expr
 
 go 1.18
 
-require github.com/stretchr/testify v1.8.4
+require (
+	github.com/agiledragon/gomonkey/v2 v2.11.0
```

Comment:
Let's not add additional dependencies.

Reply:
It's just a test dependency to address the race in the tests.  Any other preferred method for avoiding the race condition, or just leave it?

Reply:
It is a security concern. I'm also will be happy to avoid testify if possible. I understand what this is test dep, but apparently vet tools not: https://github.com/expr-lang/expr/issues/618